src/autocomplete.js

```diff
@@ -206,7 +207,7 @@ export default class Autocomplete extends Component {
 
     this.setState({ query })
 
-    const searchForOptions = !queryEmpty && queryChanged && queryLongEnough
+    const searchForOptions = showAllValues || (!queryEmpty && queryChanged && queryLongEnough)
```

Comment:
I wonder if it makes sense to fold this behaviour into just the `minLength` option? Current `minLength: 0` default does nothing compared to `minLength: 1`, it could mean "show all values" instead.